model/event/distro_event.go

```diff
@@ -62,11 +61,6 @@ func LogDistroAdded(distroId, userId string, data interface{}) {
 
 // LogDistroModified should take in DistroData in order to preserve the ProviderSettingsList
 func LogDistroModified(distroId, userId string, before, after interface{}) {
-	// Stop if there are no changes
```

Comment:
re removing this for safety -- does it seem probable that this fixed it? Or do we just want to log here if we hit this line for a bit since presumably this doesn't happen? We will want to add it in eventually i imagine.

Reply:
It does seem probable; the only other place this function is called is from REST routes that don't do any struct copying like this. Stopping + logging is a good idea, added!